### ISSUE TITLE
Non standard activity diagram notation

### ISSUE THREAD

**Issue by yongxiangng** [yongxiangng/pe#23]
![image.png](https://raw.githubusercontent.com/yongxiangng/pe/main/files/1092e281-4939-4446-b126-0dee95715bd4.png)

Arrowhead should be at the end and appear only once (I don't think it is mentioned in the notes that extra arrow heads are allowed)

Is this a plantUML limitation? If so maybe put this under out of scope i guess
